en/flight_modes/follow_me.md

```diff
@@ -8,16 +8,22 @@ author: Jimmy Johnson
 
 *Follow Me* mode allows a multicopter to autonomously follow and track another system that is broadcasting its position using the [FOLLOW_TARGET](https://mavlink.io/en/messages/common.html#FOLLOW_TARGET) MAVLink message.

```

Comment:
Are you following something if you are leading it? I suspect not, so I'm suggesting we highlight that it is holding position

```suggestion
*Follow Me* mode allows a multicopter to autonomously hold position and altitude relative to another (possibly moving) system that is broadcasting its position using the [FOLLOW_TARGET](https://mavlink.io/en/messages/common.html#FOLLOW_TARGET) MAVLink message.

```